modulefiles/gsi_wcoss2.lua

```diff
@@ -19,11 +19,5 @@ load(pathJoin("python", python_ver))
 load(pathJoin("prod_util", prod_util_ver))
 
 load("gsi_common")
-unload("crtm")
-
-pushenv("HPC_OPT", "/apps/ops/para/libs")
-prepend_path("MODULEPATH", "/apps/ops/para/libs/modulefiles/compiler/intel/19.1.3.304")
-prepend_path("MODULEPATH", "/apps/ops/para/libs/modulefiles/mpi/intel/19.1.3.304/cray-mpich/8.1.7/")
-load("crtm/2.4.0")
```

Comment:
@RussTreadon-NOAA Thanks.  CRTM 2.4.0 was installed this Monday.  We should remove this.  Thanks!!